src/js/modules/articles/actions.js

```diff
@@ -1,11 +1,55 @@
 import axios from 'axios';
 import * as t from './actionTypes';
+import STUY_SPEC_API from '../../constants';
 
-const apiUrl = 'http://api.icndb.com/jokes/random';
-
+//TODO: need a way to call the fetch function
 export const fetch = () => {
-	return {
-		type: t.FETCH_ARTICLE,
-		payload: axios.get(apiUrl),
-	};
+  return (dispatch, getState) => {
+    dispatch({ type: t.FETCH_ARTICLE_PENDING });
+    //axios.get(apiUrl, {'headers': {'X-Key-Inflection': 'camel'}}) TODO: wait until Nic uses olive branch
+    axios.get(STUY_SPEC_API+'/articles')

```

Comment:
consider string interpolation `${STUY_SPEC_API}/articles` It's minor but it's a little more readable imo